extensions/wikia/ImageReview/ImageReviewHelper.class.php

```diff
@@ -97,18 +97,18 @@ public function updateImageState( $images, $action = '' ) {
 	public function updateStatsOnAction( $action, $reviewed, $approved, $questionable, $rejected, $deleted ) {
 		switch ( $action ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a "case default" clause to this "switch" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS131)